cfn-templates/data-exports-aggregation.yaml

```diff
@@ -0,0 +1,1091 @@
+        ZipFile: |
+          import os
+          import json
+          import uuid
+
+          import boto3
+          import cfnresponse
+
+          region = os.environ['AWS_REGION']
+
+          # CUR2 only exists in us-east-1 and cn-northwest-1 regions
+          if region.startswith('cn-'):
+              region = 'cn-northwest-1'
+          else:
+              region = 'us-east-1'
+
+          client = boto3.client('cur', region_name=region)
+
+          def lambda_handler(event, context):
+
+            print(json.dumps(event))
+            reason = ""
+
+            try:
+              export = event['ResourceProperties']['Export']
+              export_name = event['ResourceProperties']['Export']['Name']
+
+              if event['RequestType'] == 'Create':
+                  res = client.put_export_definition(
+                      Export=export
+                  )
+                  print(json.dumps(res))
+              elif event['RequestType'] == 'Update':
+                  old_export_name = event['OldResourceProperties']['Export']['Name']
+                  if export["Name"] != old_export_name:
+                      res = client.create_export(
+                          Export=export
+                      )
+                      print(json.dumps(res))
+                  else:
+                      res = client.update_export(
+                          Name=old_export_name,
+                          Export=export
+                      )
+                      print(json.dumps(res))
+              elif event['RequestType'] == 'Delete':
+                  try:
+                      res = client.delete_export(
+                          Name=export_name
+                      )
+                      print(json.dumps(res))
+                  except:
+                      pass # Do not block deletion
+              else:
+                  raise Exception("Unknown operation: " + event['RequestType'])
+
+            except Exception as e:
+                reason = str(e)
+                print(e)
+            finally:
+                physicalResourceId = event.get('ResourceProperties',{}).get('Export').get('Name', None) or str(uuid.uuid1())
+                if reason:
+                    print("FAILURE")
+                    cfnresponse.send(event, context, cfnresponse.FAILED, {"Data": reason }, physicalResourceId)
+                else:
+                    print("SUCCESS")
+                    cfnresponse.send(event, context, cfnresponse.SUCCESS, {}, physicalResourceId)
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: 'W89'
+            reason: "This Lambda does not require VPC"
+          - id: 'W92'
+            reason: "One Time execution. No need for ReservedConcurrentExecutions"
+
+###########################################################################
+# Glue Database for CID framework
+###########################################################################
+  CIDDatabase:
```

Comment:
This can be a blocker. I struggle to understand the full flow if CID database is defined here